apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.ts

```diff
@@ -116,6 +117,12 @@ router.post(
       const origHash = req.body.orig_hash;
       questionInfo.title = req.body.title;
       questionInfo.topic = req.body.topic;
+      // If only a single tag is provided, it will be a string. If multiple tags are provided, it will be an array.
+      if (Array.isArray(req.body.tags)) {
+        questionInfo.tags = req.body.tags;
+      } else {
+        questionInfo.tags = [req.body.tags];
+      }
```

Comment:
Make sure to test your changes! You should check what happens when you save a question with no tags.

Reply:
Good catch! In [36eeec6](https://github.com/PrairieLearn/PrairieLearn/pull/10973/commits/36eeec610c7ced8576c82f19fa8c09767d22f110),  I have updated so that if you save a question with no tags, we do not try to store `null` and actually the property is removed from the JSON.